Templates/account/Request.liquid

```diff
@@ -1,15 +1,16 @@
 {
-  "method": "POST",
-  "headers": {
-    {%- for header in source.headers -%}
-      {{ header | as_header }},
-    {%- endfor -%}
-      "Content-Type":"application/json"
-  },
-  "body": {
-      "Account": {
-        "AccountId": "{{ source.body.accountId }}",
-        "PostalCode": "{{ source.body.customIdentifierFields.postalCode }}"
-      }
-  }
+    "method": "POST",
+    "url": "{{ config.baseUrl | append_url: config.url }}",
```

Comment:
Who will be responsible for eliminating unnesessary `/` delimiters in this expression?
Imagine if baseUrl ends with `/` and config.url starts with '/' for some reason too

Reply:
`| append_url` filter is responsible for this now.

Reply:
It is still possible to use an endpoint without any template. (In which case the message is passed over as-is)
The URL in this case is created the old way, by concatenating `baseUrl` + `url`.
So both ways are technically used.